pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -347,6 +347,8 @@ RUN HOME_DIR=/root \
  && rm -rf ${HOME_DIR}/oss_compliance* \
  && rm -rf /tmp/tmp*
 
+RUN rm -rf ~/.cache
```

Comment:
What's the reason for adding this?

Reply:
Got it. So this is a backport of something we've implemented in images with newer FW versions.